mobile/src/bundles/talent/components/preview-tabs/preview-tabs.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import { type ValueOf } from 'shared/build/index';
+
+import {
+    Text,
+    TouchableOpacity,
+    View,
+} from '~/bundles/common/components/components';
+import { TextCategory } from '~/bundles/common/enums/enums';
+import { globalStyles } from '~/bundles/common/styles/styles';
+import { type BsaBadgeStepBadgesTitle } from '~/bundles/talent/enums/enums';
+import { ProfileTab } from '~/bundles/talent/enums/enums';
+
+import { Feedbacks } from '../feedbacks/feedbacks';
+import { Project } from '../project/project';
+import { ScoresAndSkills } from '../scores-and-skills/scores-and-skills';
+import { styles } from './styles';
+
+type Tab = ValueOf<typeof ProfileTab>;
+
+type Badge = {
+    label: ValueOf<typeof BsaBadgeStepBadgesTitle>;
+    value: number;
+};
+
+//todo change to real types
+type Properties = {
+    badges: Badge[];
+    hardSkills: string[];
+    personalType: string[];
+    HRBadges: string[];
+};
+
+const PreviewTabs = ({
+    badges,
+    hardSkills,
+    personalType,
+    HRBadges,
```

Comment:
I guess you can select this data from the store, we already store steps values there

Reply:
HRBadges this is hard-coded data (in feedback)

I use props drilling from PorfilePreview, or useAppSelector here is better option?

Reply:
Let's use useAppSelector